e2e-polybft/e2e/txpool_test.go

```diff
@@ -48,17 +48,25 @@ func TestE2E_TxPool_Transfer(t *testing.T) {
 		go func(i int, to ethgo.Address) {
 			defer wg.Done()
 
-			gasPrice, err := client.GasPrice()
-			require.NoError(t, err)
-
 			txn := &ethgo.Transaction{
```

Comment:
WDYT about being explicit and set transaction type to either `TransactionLegacy` or `TransactionDynamicFee`?